src/Microsoft.Azure.ServiceBus/Management/ManagementConstants.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.ServiceBus.Management
+{
+    internal class ManagementConstants
```

Comment:
Same name used for AMQP constants. Suggest to rename to something like `ManagementClientConstants`?